short/codespaces-project-template-js/README.md

```diff
@@ -0,0 +1,105 @@
+# JavaScript Portfolio Site with GitHub Codespaces
+
+_Create, customize and deploy your own portfolio website in minutes._ ✨
+
+In this template repository we have the development environment, base project and deployment steps set and ready so that you can immediately launch the Codespace to customize your own portfolio web application with no setup.
+
+Built to be a "choose your own adventure" template repository portfolio, we have a [React](https://reactjs.org/) based web application ready for you to easily customize and deploy using only your web browser. _No web development experience required._
+
+![JavaScript profile web application](./images/js-portfolio-site.gif "JavaScript profile web application")
+
+## About Codespaces
+
+Using Codespaces, you get [Visual Studio Code](https://visualstudio.microsoft.com/?WT.mc_id=academic-77460-alfredodeza) in the cloud, using a ["development container"](https://containers.dev/). Like a locally running version of [Visual Studio Code](https://visualstudio.microsoft.com/?WT.mc_id=academic-77460-alfredodeza), the cloud version also allows you to install extensions and use a terminal.
+
+You can also configure your development container to run a specific runtime and have it boot up with your favorite extensions.
+
+## What you will learn and requirements for this workshop
+
+| **Goal**              | Description                                    |
+| ----------------------------- | --------------------------------------------------------------------- |
+| **What will you learn**       | customizing Codespaces, web development, and web application deployment              |
+| **What you'll need**          | <ul><li>[GitHub account](https://github.com)</li><li>access to [Codespaces](https://github.com/features/codespaces)</li><li> optional [Azure for Students](https://azure.microsoft.com/en-us/free/students/) or [Azure](https://azure.microsoft.com/en-us/free/search/) account for deployment</li></ul> |
```

Comment:
The azure links need to have WT links and get the localization (`en-us`) removed

Reply:
Thanks or catching this localized route! All have been removed.